test/integration/features/cmd-addons.feature

```diff
@@ -0,0 +1,158 @@
+@cmd-addons @command
+Feature: Addons command and its subcommands
+
+  Scenario: Command "addons" has "adddon" alias
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift addons" succeeds
+     Then executing "minishift addon" succeeds
+      And executing "minishift add-ons" fails
+      And executing "minishift add-on" fails
+
+  @minishift-only
+  Scenario: Default addons are installed but not enabled by default
+  Note: default addons were installed during previous commands automatically.
+     When executing "minishift addons list"
+     Then stdout should match "admin-user\s*: disabled\s*P\(0\)"
+      And stdout should match "anyuid\s*: disabled\s*P\(0\)"
+      And stdout should match "registry-route\s*: disabled\s*P\(0\)"
+      And stdout should match "xpaas\s*: disabled\s*P\(0\)"
+
+  Scenario: Uninstalling an add-on
+     When executing "minishift addons uninstall anyuid" succeeds
+     Then executing "minishift addons list" succeeds
+      And stdout should not contain "anyuid"
+
+  Scenario: Installing add-on from a folder
+  Note: working directory when executing Minishift commands is: /test/integration.
+     When file from "https://raw.githubusercontent.com/minishift/minishift/master/addons/anyuid/anyuid.addon" is downloaded into location "download/anyuid"
+      And executing "minishift addons install ../../out/integration-test/download/anyuid" succeeds
+     Then executing "minishift addons list" succeeds
+      And stdout should contain "anyuid"
+
+  Scenario: Applying add-on when Minishift is not running
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift addons apply anyuid" succeeds
+     Then stdout should contain
+      """
+      Running this command requires an existing 'minishift' VM, but no VM is defined.
+      """
+
+  Scenario: Removing add-on when Minishift is not running
+  After issue no. 1337 is resolved, please change the expected stdout to:
+  Minishift should return: "Running this command requires an existing 'minishift' VM, but no VM is defined."
```

Comment:
You mean https://github.com/minishift/minishift/issues/1337? I guess you are talking about a different issue.

Reply:
@LalatenduMohanty Should have been #1377. Going to change it.